kv/tikv/storage/commands/rollback.go

```diff
@@ -0,0 +1,70 @@
+package commands
+
+import (
+	"github.com/pingcap-incubator/tinykv/kv/tikv/storage/kvstore"
+	"github.com/pingcap-incubator/tinykv/proto/pkg/kvrpcpb"
+)
+
+type Rollback struct {
+	request *kvrpcpb.BatchRollbackRequest
+}
+
+func NewRollback(request *kvrpcpb.BatchRollbackRequest) Rollback {
+	return Rollback{request}
+}
+
+func (r *Rollback) BuildTxn(txn *kvstore.MvccTxn) error {
+	txn.StartTS = &r.request.StartVersion
+	for _, k := range r.request.Keys {
+		rollbackKey(k, txn)
+	}
+	return nil
+}
+
+func rollbackKey(key []byte, txn *kvstore.MvccTxn) error {
+	lock, err := txn.GetLock(key)
+	if err != nil {
+		return err
+	}
+	if lock == nil || lock.TS != *txn.StartTS {
+		return &LockNotFound{key}
```

Comment:
should check write here?  maybe it's already rollbacked.

we can refer to the logic of tikv release-2.0 branch.
https://github.com/tikv/tikv/blob/release-2.0/src/storage/mvcc/txn.rs#L252